Geometry/EcalCommonData/plugins/dd4hep/DDEcalBarrelNewAlgo.cc

```diff
@@ -1231,10 +1194,12 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
               const double yy(radius * sin(phi));
               const double xx(radius * cos(phi));
               ++fanOutCopy;
-              xilyLog.placeVolume(ilyFanOutLog,
-                                  fanOutCopy,
-                                  Transform3D(RotationZ(phi) * RotationY(180_deg),
-                                              Position(xx, yy, ily.vecIlyFanOutZ[ilyFO] - ilyLengthHalf)));
+              xilyLog.placeVolume(
+                  ilyFanOutLog,
+                  fanOutCopy,
+                  Transform3D(
+                      RotationZ(phi) * RotationY(180_deg),  // CLHEP::HepRotationZ(phi) * CLHEP::HepRotationY(180 * deg)
```

Comment:
Why is the CLHEP comment being added to this line?

Reply:
removed